src/plugins/intel_cpu/src/nodes/conv.cpp

```diff
@@ -1542,7 +1542,29 @@ void Convolution::redefineOutputMemory(const std::vector<VectorDims> &newOutputS
 
 MemoryDescPtr Convolution::getSumMemDesc(const primitive_desc &primitive_desc_it) {
     if (getOutputShapeAtPort(0).isDynamic()) {
-        return DnnlExtensionUtils::makeUndefinedDesc(primitive_desc_it.dst_desc(0), getOutputShapeAtPort(0));
+        // When we set input shape with range dims, real input shape maybe mismatch with output shape, we just change
+        // range min value to 1 to meet this case.
+        // For example:
+        // Output shape = {1, 160, {128, 256}, {128, 256}}
+        // Real input shape = {1, 160, 1, 1}
```

Comment:
Lets rename all instances of `real` to `sum` for better understanding.

Reply:
Done